R/modules.R

```diff
@@ -0,0 +1,57 @@
+# Creates an object whose $ and $<- pass through to the parent
+# session, unless the name is matched in ..., in which case
+# that value is returned instead. (See Decorator pattern.)
+createSessionProxy <- function(parentSession, ...) {
+  e <- new.env(parent = emptyenv())
+  e$parent = parentSession
```

Comment:
Oh yeah! Follow your heart, Joe! :wink: All assignments shall be created **equal**! (Coincidently I just started this war on Twitter a couple of hours ago today)